.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+	extends: '@rowanmanning/eslint-config/legacy',
+	globals: {
+		define: true
+	},
+	rules: {
+		'id-length': 'off',
```

Comment:
Where are we using short identifiers?

Reply:
Oh yeah I added that in temporarily. Will remove